server/src/functions/product/handler.ts

```diff
@@ -0,0 +1,14 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+
+const getProductsById: ValidatedEventAPIGatewayProxyEvent<null> = async () => {
```

Comment:
From my opinion would be better return certain item after find from the whole array, not just the first item. Try to pass id as a argument. 
![image](https://user-images.githubusercontent.com/73526360/221541154-6e3aab27-e7c6-43fa-bb1d-9ffb328c61c6.png)
Yeap we use mock but it looks strange:
![image](https://user-images.githubusercontent.com/73526360/221541257-99ca217d-37fe-4f1f-bec3-4b7221ae1d5e.png)

Reply:
I didn't see a point in doing that with mock data. Fixed, now should work